cmd/list.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"github.com/spf13/cobra"
+	"github.com/sector-f/jh_extract/nvc"
+)
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Manipulate nvc files",
+	Long: `based off jh_extract.py`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		reader, openErr := os.Open(args[0])
+		if (openErr != nil) {
+			fmt.Println(openErr)
+			os.Exit(1)
+		}
+		entries, listErr := nvc.ListEntries(reader)
+		if (listErr != nil) {
```

Comment:
No parentheses needed in boolean expressions

Reply:
:boom: